style.css

```diff
@@ -69,22 +71,85 @@ p {
   color: #344563;
 }
 
+.hero {
+  display: flex;
+  flex-direction: column;
+  padding: 20.6rem 2.4rem 0 2.4rem;
+}
+
 .hero-text {
   font-size: 1.6rem;
 }
 
 .container {
-  background-image: url(images/Background-header.png);
+  background-color: var(--color-secondary);
+  background-image: url(images/header-shapes\ mobile.svg);
   background-repeat: no-repeat;
   background-size: cover;
+  border-bottom-left-radius: 5rem;
+}
+
+@media screen and (min-width: 76.8rem) {
```

Comment:
- [ ] The transition point from mobile to desktop view should be 768px wide. Kindly change that to match the project requirement.
 - [OPTIONAL] Since this is a mobile-first approach, it is recommended to add all the code to style the mobile version on top of the media query, and then you only need to add one media-query transition point in your CSS file(for the 768px or wide) and then the code for the desktop below it.
 - 
<img width="949" alt="image" src="https://user-images.githubusercontent.com/61919617/206235226-8d69f3e5-8cf6-4ff4-aff1-473ec29be7e4.png">